lib/number_of_parameters/models/number_of_parameters_parameters.dart

```diff
@@ -0,0 +1,19 @@
+/// A data model class that represents the "number of parameters" input
+/// parameters.
+class NumberOfParametersParameters {
+  /// Maximum number of parameters
+  final int maxParameters;
+
+  static const _defaultMaxParameters = 2;
```

Comment:
Might make sense co collect all defaults in one place later on. For now let's roll with local constantw